### ISSUE TITLE
How to position the style instructions?

### ISSUE THREAD

**Issue by Frolipon** [maizzle/framework#729]
- Maizzle Version: 4.0.2
- Node.js Version: 16.15.0

Hello,

I'm trying to use Maizzle to create emails for another tool (Mosaico).
So far, I'm stuck at one point: I need to write instruction for Mosaico **at the end** of the style attribute.

Example of expected code:
```
<p style="color: #000000;-ko-color: @[colorTheme eq 'white' ? white.fontColorVar : black.fontColorVar];">
 My text
</p>
```
Of course, `-ko-color: @[colorTheme eq 'white' ? white.fontColorVar : black.fontColorVar]` is for Mosaico.

Is there a way to position specific instruction at the end of the style attribute?
I have tried:
```
<p class="text-black" style="-ko-color: @[colorTheme eq 'white' ? white.fontColorVar : black.fontColorVar];">
 My text
</p>
```

but then I get in output:
```
<p style="-ko-color: @[colorTheme eq 'white' ? white.fontColorVar : black.fontColorVar];color: #000000;">
 My text
</p>
```

And because of the characters used for this code, I didn't managed to use it like:
```
<p class="text-black [-ko-color:_@[colorTheme_eq_'white'_?_white.fontColorVar_:_black.fontColorVar]]">
 My text
</p>
```
The main problem being the `:`,`[` and `]`, I tried to escape them with `\:` or `\[` but it doesn't work.

Is there any way to tell Tailwind to put the content used in the style tag after the style coming from the tailwind classes? So I could get:
use
```
<p class="text-black" style="-ko-color: @[colorTheme eq 'white' ? white.fontColorVar : black.fontColorVar];">
```
and get
```
<p style="color: #000000;-ko-color: @[colorTheme eq 'white' ? white.fontColorVar : black.fontColorVar];">
```

**Comment by cossssmin** [comment]
I think the easiest way would be to disable CSS inlining, though that will cause trouble in Outlool for Windows…

Another idea would be to use the Replace Strings Transformer to add that inline style after all compilation is done. 

https://maizzle.com/docs/transformers/replace-strings

You could also use the [afterTransformers event](https://maizzle.com/docs/events#aftertransformers) with a regex and add it in a similar fashion.

**Comment by Frolipon** [comment]
Disabling CSS inlining is not really an option for us because of Outlook and some other small email clients.

I have found kind of a workaround: I have found that I only need my specific code to be behind the CSS properties it will manage, so not at the end. With that, I have accepted that I will write always in the style attribute my CSS properties and the Mosaico code that goes with it, and I can use Maizzle/Tailwind for the rest, like:
`<p class="text-center" style="color: #000000;-ko-color: @[colorTheme eq 'white' ? white.fontColorVar : black.fontColorVar];">`

I have checked the Replace Strings Transformers. I'm still not sure of the most efficient way to use it in my case, but it is certainly a very useful tool, thanks a lot for pointing it out!
